Kitodo-API/src/main/java/org/kitodo/api/rulesetreader/RulesetReaderInterface.java

```diff
@@ -33,45 +33,45 @@
 
     /**
      * Get all possible acquisition stages.
-     * 
+     *
      * @return A Collection of all acquisition stages
      */
     Collection<String> getAllAcquisitionStages();
 
     /**
-     * Get all metadata, which can be added for a specific division.
-     * 
+     * Get all meta-data, which can be added for a specific division.
+     *
      * @param rulesetRequestData
      *            The needed data for retrieval.
-     * @return A {@link Map} collection of addable metadata that contains
+     * @return A {@link Map} collection of addable meta-data that contains
      *         {@link String} as id and {@link String} as translation
      */
     Map<String, String> getAddableMetadataForDivision(RulesetRequestData rulesetRequestData);
 
     /**
-     * Gets the sorted visible metadata.
-     * 
+     * Gets the sorted visible meta-data.
+     *
      * @param rulesetRequestData
      *            The needed data for retrieval.
-     * @return a Collection of Metadata.
+     * @return a Collection of Meta-data.
```

Comment:
Here it is even more pointless - Collection of Metadata - means collection of Metadata class objects :)